src/delivery-options/DeliveryOptions.vue

```diff
@@ -111,20 +113,25 @@ export default {
           }
 
           this.showDeliveryOptions = true;
-          this.listeners.update();
+          this.fakeShowDeliveryOptions = true;
           document.addEventListener(EVENTS.UPDATE_DELIVERY_OPTIONS, this.listeners.update);
+          this.listeners.update();
         },
         hide: () => {
           this.$configBus.exportValues = {};
           this.fakeShowDeliveryOptions = false;
 
-          const listenForLastUpdate = () => {
+          const onLastUpdate = () => {
             this.showDeliveryOptions = false;
             document.removeEventListener(EVENTS.UPDATE_DELIVERY_OPTIONS, this.listeners.update);
-            document.removeEventListener(EVENTS.UPDATED_DELIVERY_OPTIONS, listenForLastUpdate);
+            document.removeEventListener(EVENTS.UPDATED_DELIVERY_OPTIONS, onLastUpdate);
+
+            clearTimeout(lastUpdateTimeout);
```

Comment:
Dit ziet er heel erg uit alsof je debounce na doet. Die gebruik je hieronder ook al, past dit hier ook in?

Reply:
Niet echt, wat ik probeer te doen is die timeout als een fallback te laten draaien, voor het geval die event listener niet getriggered wordt.

Reply:
Ah oké, ik was gepranked door de `clearTimeout`. Die is niet nodig als de timer afgaat, maar wel als de functie ook door events worden gebruikt.